examples/Demo/FluentUI.Demo.Client/Documentation/Components/Switch/Examples/SwitchDisabled.razor

```diff
@@ -0,0 +1,7 @@
+﻿<FluentSwitch Label="Checked and disabled" Disabled="true" @bind-Value="@value1" />
+<FluentSwitch Label="Unchecked and disabled" Disabled="true" @bind-Value="@value2" />
+
```

Comment:
Is it possible to use `ReadOnly`?

Reply:
I can add a Readonly example and tests but it does not prevent from switching it, it is just a little bit greyer.

![image](https://github.com/user-attachments/assets/5ce922ed-184f-4ce7-bc02-c9a82ca4cc34)

Compare with disabled:

![image](https://github.com/user-attachments/assets/38c84c52-2de8-449d-8cd1-b6595d4af40d)

Reply:
With ReadOnly the component cannot be changed. You need to find a way to do that. Maybe using a CSS style `pointer-events: none;` and 
`tabindex="-1"` (only if the `AdditionalAttributes` don't contain this key.

Reply:
`pointer-events: none;` added if `ReadOnly` is true (prevents from clicking on it).